fitsio/util.py

```diff
@@ -134,9 +134,22 @@ def array_to_native(array, inplace=False):
     return output
 
 
+if numpy.lib.NumpyVersion(numpy.__version__) >= "2.0.0":
+    copy_if_needed = None
```

Comment:
What does `None` mean?  I don't see that mentioned in the release notes https://numpy.org/devdocs/release/2.0.0-notes.html

Reply:
I'm just following the `scipy` shim code and it does seem to work.

Reply:
See my comments on the main thread. I believe we want to keep False.

Reply:
Then all the tests crash.